pom.xml

```
@@ -32,7 +32,7 @@
     </scm>
 
 	<properties>
-		<java-version>1.7</java-version>
+		<java-version>1.8</java-version>
```

As far as I can see there is not features of 1.8 used so I would leave it at 1.7 for better backward compatibility.
```suggestion
		<java-version>1.7</java-version>
```